recipes-ni/ni-rtfeatures/files/rtfeatures.rules

```diff
@@ -0,0 +1,5 @@
+# create symlink to establish a uniform path to nirtfeatures
+DRIVER=="nirtfeatures",RUN+="/bin/ln -sf /sys%p /dev/nirtfeatures"
+
+#give all users read/write access to CPLD-created /dev/input/event?
+SUBSYSTEM=="input", ENV{ID_PATH}=="acpi-NIC775D:??", KERNEL=="event?", MODE="0666"
```

Comment:
This is CPLD related, but not `rtfeatures` related. Maybe it does not belong here, but it does not feel like it belongs in `localextra.rules` either. I can be convinced to move it back to `localextra.rules`.

Reply:
Is it required for the other content in this package to work?

If not, I would agree that it would be better in a specific package; but maybe not this one?

Reply:
It is not required for rtfeatures to work, but keeping the grouping the CPLD and rtfeatures together makes sense.